go.mod

```diff
@@ -9,16 +9,23 @@ require (
 	github.com/google/go-cmp v0.5.6
 	github.com/google/go-containerregistry v0.6.0
 	github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
-	github.com/pivotal/kpack v0.3.1
+	github.com/pivotal/kpack v0.3.2-0.20211004190409-3e4bdcd83b29
 	github.com/pkg/errors v0.9.1
 	github.com/sclevine/spec v1.4.0
 	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a
+	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	k8s.io/api v0.20.7
-	k8s.io/apimachinery v0.20.7
-	k8s.io/client-go v0.20.7
-	knative.dev/pkg v0.0.0-20210819054404-bda81c029160
+	k8s.io/api v0.21.3
+	k8s.io/apimachinery v0.21.3
+	k8s.io/client-go v0.21.3
+	knative.dev/pkg v0.0.0-20210902173607-844a6bc45596
```

Comment:
Do we need a direct dep on knative?

Reply:
i removed it and go mod tidy added it right back

Reply:
looks like i added something to the commit that shouldn't be there

Reply:
Never mind, it was added in fd1a7c926552b63446bb5d4669268cc3bbf6e13a. Looks like we actually use it

Reply:
`go mod why -m knative.dev/pkg`

Can we drop this file https://github.com/vmware-tanzu/kpack-cli/blob/main/pkg/commands/waiter.go#L22?

Reply:
potentially, but i think that might be outside the scope of this pr